birdvoxpaint/birdvoxpaint.py

```diff
@@ -1,76 +1,110 @@
+from functools import partial
+
 import librosa
-from librosa.util.exceptions.ParameterError
+import librosa.display
 import numpy as np
 
+import tqdm
+import joblib
+
+from . import util
+from .indices import *
+from .display import *
+
+MAX_JOBS = 40
+
+def transform(filename=None, y=None, sr=None,
+              frame_length=2048, hop_length=512,
+
+              n_mels=256, fmin=None, fmax=None,
+              time_constant=0.4, use_pcen=True, use_mels=True,
```

Comment:
i don't think that `use_pcen` should be a boolean flag. ideally, we'd like to be able to have indices from both PCEN and non-PCEN versions of the spectrogram. i would prefer to make an index named `average_pcen` (or `max_pcen` if you prefer) that we can include from `indices`. But that will require a bit of code introspection to pass `zf` into `zi` and parallelize properly.

Reply:
@beasteers would you be OK with updating this and make separate indices for PCEN?

Reply:
Oops, I already did that. I just forgot to resolve the convo